docs/usage.md

```diff
@@ -0,0 +1,94 @@
+# Arcadia-Science/metagenomics Usage
+
+This page provides documentation on how to use the Arcadia-Science/metagenomics pipeline.
+
+## Input specifications
+
+This pipeline processes metagenomic reads in FASTQ format from either Illumina or Nanopore technologies. **Important note**: This pipeline separately processes Illumina or Nanopore metagenomes and does not perform hybrid assembly, polishing of Nanopore assemblies with Illumina reads, or scaffolding with Nanopore reads.
```

Comment:
I think this could be on the main README somewhere as well if you can find a place that it fits well. I think this is probably a pretty important statement